src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const values in robot) {
+    const newValue = robot[values];
```

Comment:
you can create variable const newValue = robot [values] after return null, it'd be more appropriately